source/fundamentals/atlas-search.txt

```diff
@@ -510,12 +510,33 @@ an ``establishedYear`` value greater than 1980 and less than 2020.
 
 The search returns the following results:
 
-.. code-block:: json
+.. code-block:: text
 
    { "_id" : 3, "make" : "PRS", "description" : "High-end guitars known for their quality.", "establishedYear" : 1985, "in_stock" : true, "rating" : 9 }
    { "_id" : 4, "make" : "Kiesel", "description" : "Quality guitars made only for custom orders.", "establishedYear" : 2015, "in_stock" : false, "rating" : null }
    { "_id" : 6, "make" : "Strandberg", "description" : "Modern guitars known for their headless models.", "establishedYear" : 1982, "in_stock" : false, "rating" : null }
 
+To search for documents in which the value of a specified field is within a
+range of strings, you must first create a :atlas:`token index
```

Comment:
S: consider describing what a range of strings is

Reply:
Added some text after the example the explains it